src/libs/protocols/protos/image.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package grpc;
+
+// Image defines the public APIs for managing images.
+service Image {
+    // PullImage pulls an image with authentication config.
+    rpc PullImage(PullImageRequest) returns (PullImageResponse) {}
+}
+
+message PullImageRequest {
+    // Image name (e.g. docker.io/library/busybox:latest).
```

Comment:
+1 for giving an example, but it would be useful to add a comment either explaining the full format of `image`, or providing a canonical URL for a doc that explains it. For example:

- Can I use an alternative to `docker.io`?
- is `:$tag` optional? If not specified, which version is pulled?
- `latest` is useful for users, but not helpful for problem determination (_"what was the value of 'latest' 2 weeks ago?"_) Hence, Is `latest` expanded to an actual version number and logged at pull time for auditing purposes?

Reply:
Ok, will add some link for reference to explaining the full format of docker image.